### ISSUE TITLE
Leaderboard: Add zero-shot filter to leaderboard

### ISSUE THREAD

**Issue by KennethEnevoldsen** [embeddings-benchmark/mteb#1372]
Add zero-shot generalization to model metadata. This would allow us to filter MTEB models.

**Comment by Muennighoff** [comment]
Cc'ing @jhyuklee & Xiaoqi Ren (can't find the GitHub handle) as you were very interested in this! Hopefully will be there with a new leaderboard soon, any thoughts from you are ofc very welcome

I recall that you mentioned that https://huggingface.co/BAAI/bge-en-icl has the ArguAna test set in their training dataset they released and relatedly oddly high ArguAna Retrieval scores, so that's one to remember to flag in our metadata for the new leaderboard

**Comment by KennethEnevoldsen** [comment]
Closing in favor of #1556
